network.go

```diff
@@ -75,6 +78,53 @@ func (n *network) Type() string {
 	return n.driver.Type()
 }
 
+func (n *network) Key() []string {
+	return []string{"network", string(n.id)}
+}
+
+func (n *network) Value() []byte {
+	b, err := json.Marshal(n)
```

Comment:
Are you marshalling the whole object as value? I though you are going to use the below func `MarshalJSON` and `UnmarshalJSON`